frontend/src/views/flight-details.js

```diff
@@ -0,0 +1,23 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+import airplane from "../images/airplane.png";
+import { Button } from "../components/ui";
+
+function FlightDetails() {
+  return (
+    <div>
+      <img src={airplane} css={{ margin: "auto", display: "block" }} />
+      <ul css={{ padding: "1em", border: "1px solid black", margin: "1em 0" }}>
+        <li>Fliht number: 311</li>
+        <li>Capacity: 100</li>
```

Comment:
You can already bring this info from the store

Reply:
Yes.  I will do it! Thanks!